llvm/test/MC/AsmParser/directive_base64.s

```diff
@@ -0,0 +1,28 @@
+# RUN: not llvm-mc -triple i386-unknown-unknown %s | FileCheck %s
+# RUN: not llvm-mc -triple i386-unknown-unknown -o /dev/null %s 2>&1 | FileCheck %s --check-prefix=CHECK-ERROR
+
+        .data
+# CHECK: TEST0:
+# CHECK: .byte 0
+TEST0:
+        .base64 "AA=="
```

Comment:
Should have consecutive `.base64` lines to ensure that the newline is correctly consumed.

Use `CHECK-NEXT:` whenever applicable

Reply:
Thanks! Done.